chain/jsonrpc/src/lib.rs

```diff
@@ -233,6 +232,8 @@ impl JsonRpcHandler {
                     }))
                 })?
             }
+            NetworkClientResponses::TxStatus(tx_result) => serde_json::to_value(tx_result)
+                .map_err(|err| RpcError::server_error(Some(err.to_string()))),
```

Comment:
Return `ExecutionErrorView`, nearlib depends on the typed error types.

Reply:
happens to see a similar issue in #1544 if you change this can you change it btw?
![image](https://user-images.githubusercontent.com/13259400/67909087-3ad21700-fb3b-11e9-953e-2b4e22e1ae84.png)